var/spack/repos/builtin/packages/mesa/package.py

```diff
@@ -34,6 +34,8 @@ class Mesa(AutotoolsPackage):
 
     version('12.0.3', '60c5f9897ddc38b46f8144c7366e84ad')
 
+    variant('nodri', default=False, description='Disable DRI and EGL support')
```

Comment:
Can you make this positive? Name it `dri`, and say that it enables DRI and EGL support.